policybot/pkg/storage/store.go

```diff
@@ -16,8 +16,13 @@ package storage
 
 import (
 	"io"
+
+	google_spanner "cloud.google.com/go/spanner"
```

Comment:
This is a generic spanner-free interface so that storage can be replaced over time, so let's keep all spanner stuff in the implementation only.

Reply:
Same as the answer question, the question becomes it's hard to determine where we put SQL part, that involves both Spanner specifics and business logic specifics.

Reply:
Resolved in another thread.